component/authenticator/src/main/java/org/wso2/carbon/identity/application/authenticator/totp/util/TOTPUtil.java

```diff
@@ -520,6 +522,11 @@ public static int getWindowSizeFromRegistry(String tenantDomain, AuthenticationC
      */
     public static boolean isEnrolUserInAuthenticationFlowEnabled(AuthenticationContext context) {
 
+        Map<String,String> runtimeParams = totpAuthenticator.getRuntimeParams(context);
+        boolean TOTPAdaptiveScriptEnabled=true;
```

Comment:
cant get an idea that the enrollment is the feature which is eing enabled and disabled.